### PR TITLE
Add unit test for AddPhotoStep component

### LINKED ISSUE #97: (SP: 1) Write unit test for "AddPhotoStep" container
### Container unit test
Unit test for "**AddPhotoStep**" container
Scenaries descriptions:
- [ ] Container should rendered
- [ ] Check if the buttons passed in props is in the document

[Link to component](https://github.com/Space2Study-UA-1125/frontEnd
/blob/develop/src/containers/tutor-home-page/add-photo-step/AddPhotoStep.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by iaremchuk**
Unit test for "AddPhotoStep" container
Scenaries descriptions:
- Container should rendered
- Check if the buttons passed in props is in the document

Untitled design-105

### CODE REVIEW

**Comment by OliaHrytsak** [comment]
I guess that it isn't necessary to add data-testId for each component
You can use 'findByText' for box and for button, without adding data-testId to the conponent code!

**Review by VictoriaRodik — APPROVED** [review]

**Review by iva-stasia — APPROVED** [review]

**Review by Wiret02 — APPROVED** [review]

**Review by VictoriaRodik — APPROVED** [review]

**Review by holus-bolus — APPROVED** [review]

**Review by Wiret02 — APPROVED** [review]

**Review by ashfox21 — COMMENTED** [review]
The test that checks whether the elements passed in props are rendered also confirms that the elements are rendered correctly. 
It is also not necessary to add testid in the test file, as it can be checked using the element passed in the props.

File: src/containers/tutor-home-page/add-photo-step/AddPhotoStep.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { style } from '~/containers/tutor-home-page/add-photo-step/AddPhotoStep.
 
 const AddPhotoStep = ({ btnsBox }) => {
   return (
-    <Box sx={style.root}>
+    <Box data-testid='AddPhotoStepContainer' sx={style.root}>
```
Comment (vlad-khrychov) [inline src/containers/tutor-home-page/add-photo-step/AddPhotoStep.jsx]: Agree with @OliaHrytsak, use `findByText` or mock `Box` component in test.

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: src/containers/tutor-home-page/add-photo-step/AddPhotoStep.jsx (hunk 1 above)
Comment (iaremchuk) [reply src/containers/tutor-home-page/add-photo-step/AddPhotoStep.jsx]: Mocked the Box component, the test result is as followed:

 ✓ tests/unit/containers/tutor-home-page/add-photo-step/AddPhotoStep.spec.jsx  (1 test) 21ms
SonarQube report written to /Users/olgaiaremchuk/Desktop/spaceTostudy/frontEnd/test-report.xml
Test Files  1 passed (1)
Tests  1 passed (1)
Start at  01:09:39
Duration  1.29s

**Review by vlad-khrychov — APPROVED** [review]

**Pull request closed without merging**